src/MizarData/include/MizarData/MizarData.h

```diff
@@ -28,7 +30,10 @@ class MizarData : public orbit_capture_client::AbstractCaptureListener<MizarData
  public:
   virtual ~MizarData() = default;
 
-  [[nodiscard]] std::optional<std::string> GetFunctionNameFromAddress(
```

Comment:
I can't add a comment above, but could you add a class level comment for the MizaData class?